.gitignore

```diff
@@ -30,5 +30,4 @@ Referenced Packages
 *.sublime-workspace
 **/.sfdx/
 **/.vscode/
-**/.idea/
-/config/
+**/.idea/
```

Comment:
@davisagli @jjbennett Do you think we need to add the ignore back for /config/?

Reply:
@force2b - if you don't mind. I know it is no longer supported but it still works :)